.github/workflows/run-unit-tests.yml

```diff
@@ -15,6 +15,9 @@ jobs:
       SUPABASE_SERVICE_ROLE_KEY: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZS1kZW1vIiwicm9sZSI6InNlcnZpY2Vfcm9sZSIsImV4cCI6MTk4MzgxMjk5Nn0.EGIM96RAZx35lJzdJsyH-qQwv8Hdp7fsn3W0YpN81IU
       GOOGLE_MAPS_API_KEY: ''
       VOTER_REGISTRATION_REPO_ENCRYPTION_KEY: 'DvViBWSQfwFGSetPOVbIWZrMXYJh4wTVSE/+1QI/VTI='
+      CRYPTO_KEY_COOKIES: 'V8/+Dd0HQ2RLKur/oeFel2uLDSi7B1gXp6TjQVp8iOk='
+      CRYPTO_KEY_REGISTRATION: DvViBWSQfwFGSetPOVbIWZrMXYJh4wTVSE/+1QI/VTI=
+      CRYPTO_KEY_COOKIE: V8/+Dd0HQ2RLKur/oeFel2uLDSi7B1gXp6TjQVp8iOk=
```

Comment:
^ Are all of these crypto keys necessary now? Shouldn't it just be one VOTER_REGISTRATION_REPO_ENCRYPTION_KEY and one for cookies?